libcxx/include/tuple

```diff
@@ -223,12 +224,13 @@ public:
               "Attempted to default construct a reference element in a tuple");}
 
     template <class _Tp,
-              class = _EnableIf<
+              _EnableIf<
                   _And<
                       _IsNotSame<__uncvref_t<_Tp>, __tuple_leaf>,
                       is_constructible<_Hp, _Tp>
-                    >::value
-                >
+                    >::value,
+                  bool
+                > = false
```

Comment:
I'm really curious why this change was needed.

Reply:
I've seen less SFINAE issues with MSVC when using value assignment rather than types. This may have been done out of an abundance of caution.